lib/monitoring/custom/CustomMonitoring.ts

```diff
@@ -304,7 +310,10 @@ export class CustomMonitoring extends Monitoring {
   private getAllWidgets(summary: boolean): IWidget[] {
     const filteredMetricGroups = summary
       ? this.metricGroups.filter(
-          (group) => group.metricGroup.important ?? false
+          (group) =>
+            group.metricGroup.important ??
+            group.metricGroup.addToSummaryDashboard ??
```

Comment:
`addToSummaryDashboard` should probably take precedence over `important` if present. Should have tests around that too.

Reply:
It makes sense.
Regarding the tests, do you mean adding a new test for case `important: false, addToSummaryDashboard: true` right?

Reply:
> Regarding the tests, do you mean adding a new test for case `important: false, addToSummaryDashboard: true` right?

Yup.